lib/nexmo_developer/app/controllers/dashboard_controller.rb

```diff
@@ -197,7 +197,7 @@ def coverage_from_files
 
       parts.insert(1, 'top-level') if parts.count < 3
 
-      parts = [parts[0], parts[1], parts[2..-1].join('/')] if parts.count > 3
+      parts = [parts[0], parts[1], parts[2..].join('/')] if parts.count > 3
```

Comment:
why is this necessary?

Reply:
its a rubocop cop, we can disable the cop if you'd prefer?